# **Metagenomics analysis**

Metagenomics is the study of genetic material recovered directly from environmental samples, offering insights into the diversity and functions of microbial communities. Epi2Me, a cloud-based platform by Oxford Nanopore Technologies, simplifies the analysis of metagenomic data. Below is a comprehensive guide to analyzing metagenomics using Epi2Me.

Software for metagenomics analysis

![](img/logo/60506060.png){width="170"}

![](img/logo/Docker_logo.svg.png){width="355"}

-   Epi2Me lab

-   Docker

**Materials:**

-   Processed data (.fasq or .bam)

-   Sample sheet (Optional)

### **Guide to Analyzing Metagenomics Using Epi2Me**

**Create an Account**

-   Register for an account at [Epi2Me](https://epi2me.nanoporetech.com/)

**Upload Data**:

-   Log in to your Epi2Me account.

    ![](img/Screenshot%202568-01-25%20at%2012.34.08.png)

## **Select the Appropriate Workflow**

Epi2Me provides several workflows tailored to specific applications. For metagenomics, the most relevant workflows are:

![](img/Screenshot%202568-01-25%20at%2012.35.37.png)

Launch workflow

![](img/Screenshot%202568-01-25%20at%2012.36.17.png)

**Run the Workflow**:

-   Upload your data (fastq or bam files)

    ![](img/Screenshot%202568-01-25%20at%2012.43.41.png)

<!-- -->

-   After uploading your data, choose the workflow and start the analysis.

-   Monitor progress via the Epi2Me dashboard.

### Get the report

Example: [Report](/wf-metagenomics-report.html)
